config/routes.rb

```diff
@@ -2,6 +2,8 @@
   root "static_pages#index"
   get "my_songs", to: "users#my_song"
   get "my_albums", to: "users#my_album"
+  put "unaccept/:id(.:format)", :to => "admin/lyrics#unaccept", :as => :unaccept
```

Comment:
sửa cú pháp đi em -_-

Reply:
put "unaccept/:id(.:format)", to: "admin/lyrics#unaccept", as: :unaccept